controllers/oauth.js

```diff
@@ -90,12 +111,11 @@ async function confirmed(req, res, next, client = FxAOAuthClient) {
       }
     );
     req.session.user = verifiedSubscriber;
-
-    return res.redirect("/user/dashboard");
+    return res.redirect(returnURL.pathname + returnURL.search);
   }
   // Update existing user's FxA data
   await DB._updateFxAData(existingUser, fxaUser.accessToken, fxaUser.refreshToken, fxaProfileData);
-  return res.redirect("/user/dashboard");
+    res.redirect(returnURL.pathname + returnURL.search);
```

Comment:
Should this have a `return`?

Reply:
Nevermind. - https://github.com/mozilla/blurts-server/pull/1602/files#diff-075ff2d4bc68f5a42b68daf27075ab5bL97